web_app/utils/icecube_setup.py

```diff
@@ -0,0 +1,366 @@
+        "abbreviation": "SU",
+        "is_US": False,
+        "region": _EUROPE,
+    },
+    "Stony-Brook": {
+        "cite": "Dept. of Physics and Astronomy, Stony Brook University, Stony Brook, NY 11794-3800, USA",
+        "abbreviation": "SBU",
+        "is_US": True,
+        "region": _US_CAN,
+    },
+    "Sungkyunkwan-Physics": {
+        "cite": "Dept. of Physics, Sungkyunkwan University, Suwon 16419, Korea",
+        "abbreviation": "SKKU",
+        "is_US": False,
+        "region": _ASIA_PAC,
+    },
+    "Sungkyunkwan-Basic-Science": {
+        "cite": "Institute of Basic Science, Sungkyunkwan University, Suwon 16419, Korea",
+        "abbreviation": "SKKU",
+        "is_US": False,
+        "region": _ASIA_PAC,
+    },
+    "Texas-Arlington": {
+        "cite": "Dept. of Physics, University of Texas at Arlington, 502 Yates St., Science Hall Rm 108, Box 19059, Arlington, TX 76019, USA",
+        "abbreviation": "UTA",
+        "is_US": True,
+        "region": _US_CAN,
+    },
+    "UC-Berkeley": {
+        "cite": "Dept. of Physics, University of California, Berkeley, CA 94720, USA",
+        "abbreviation": "UCB",
+        "is_US": True,
+        "region": _US_CAN,
+    },
+    "UC-Irvine": {
+        "cite": "Dept. of Physics and Astronomy, University of California, Irvine, CA 92697, USA",
+        "abbreviation": "UCI",
+        "is_US": True,
+        "region": _US_CAN,
+    },
+    "UCLA": {
+        "cite": "Department of Physics and Astronomy, UCLA, Los Angeles, CA 90095, USA",
+        "abbreviation": "UCLA",
+        "is_US": True,
+        "region": _US_CAN,
+    },
+    "Uppsala": {
+        "cite": "Dept. of Physics and Astronomy, Uppsala University, Box 516, S-75120 Uppsala, Sweden",
+        "abbreviation": "UU",
+        "is_US": False,
+        "region": _EUROPE,
+    },
+    "UW-Madison-Astronomy": {
+        "cite": "Dept. of Astronomy, University of Wisconsin–Madison, Madison, WI 53706, USA",
+        "abbreviation": "UW",
+        "is_US": True,
+        "region": _US_CAN,
+    },
+    "UW-Madison-WIPAC": {
+        "cite": "Dept. of Physics and Wisconsin IceCube Particle Astrophysics Center, University of Wisconsin–Madison, Madison, WI 53706, USA",
+        "abbreviation": "UW",
+        "is_US": True,
+        "region": _US_CAN,
+    },
+    "UW-River-Falls": {
+        "cite": "Dept. of Physics, University of Wisconsin, River Falls, WI 54022, USA",
+        "abbreviation": "UWRF",
+        "is_US": True,
+        "region": _US_CAN,
+    },
+    "Wuppertal": {
+        "cite": "Dept. of Physics, University of Wuppertal, D-42119 Wuppertal, Germany",
+        "abbreviation": "WUPPERTAL",
+        "is_US": False,
+        "region": _EUROPE,
+    },
+    "Yale": {
+        "cite": "Dept. of Physics, Yale University, New Haven, CT 06520, USA",
+        "abbreviation": "YALE",
+        "is_US": True,
+        "region": _US_CAN,
+    },
+}  # type: Dict[str, InstitutionMeta]
```

Comment:
We should figure out how to not duplicate this list across projects.

Reply:
have you seen https://github.com/WIPACrepo/keycloak-rest-services/issues/3?

Reply:
https://github.com/WIPACrepo/keycloak-rest-services/pull/4